README.md

```diff
@@ -17,11 +17,15 @@ User account service for MuckRock and DocumentCloud
 2. Enter the directory - `cd squarelet`
 3. Run the dotenv initialization script - `python initialize_dotenvs.py`
 This will create files with the environment variables needed to run the development environment.
-4. Start the docker images - `inv up`
-This will build and start all of the docker images using docker-compose.  It will bind to port 80 on localhost, so you must not have anything else running on port 80. The invoke tasks specify the `local.yml` configuration file for docker-compose.  If you would like to run docker-compose commands directly, set the environment variable `export COMPOSE_FILE=local.yml`.
-5. Set `dev.squarelet.com` to point to localhost - `sudo echo "127.0.0.1   dev.squarelet.com" >> /etc/hosts`
-6. Enter `dev.squarelet.com` into your browser - you should see the Squarelet home page.
-
+4. You may need to provide valid testing values for `STRIPE_PUB_KEYS`, `STRIPE_SECRET_KEYS` and set `STRIPE_WEBHOOK_SECRETS=None` from the MuckRock team (multiple values are comma separated only, no square braces) 
+      - You must always fully `docker-compose down` or Ctrl-C each time you change a `.django` file of a docker-compose session for it to take effect (as far as I know).
```

Comment:
I would say for any `.env` file.  I would also remove the weasel words - 
> You will need to provide
Remove as far as I know - you do need to bring the docker container down and up again for `.env` changes to take affect,

Reply:
Fixed

Reply:
But not quite sure what you meant by "for any .env file" — I think these values are only needed for Squarelet, not DocumentCloud's .django files?